src/content/docs/style-guide/images/embed-videos.mdx

```diff
@@ -32,14 +32,15 @@ Write introductory text above the embedded video or add a video caption after it
 
 Besides the docs site, Wistia hosts videos for [learn.newrelic.com](https://learn.newrelic.com/). Wistia is a good place to host videos you create as a writer, as well as videos we receive from other teams.
 
-To use Wistia to host videos:
-
-1. Log on to [Wistia.com](https://wistia.com).
-2. Click **Projects** in the left navigation pane.
-3. In the center pane, page through until you find **docs.newrelic.com Videos**.
-4. Click **Add > Upload**.
-5. Click **Embed &#38; Share**.
-6. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:
+All writers should have admin-level access to Wistia so that they can upload, manage, and track the performance of videos. If an internal contributor wants to add a video, our team will upload and manage it on their behalf. We do not want to provide contributor access to Wistia to those outside of our team.
+
+To upload videos to Wistia:
+
+1. Go to [Wistia.com](https://app.wistia.com/login/sso/new), type `newrelic`, and click **Continue with SSO**.
+2. Click **Content library > Content** and click **docs.newrelic.com Videos**.
+3. Click **Add > Upload**. If you don't see this option, you can request collaborator permissions from #help-documentation.
```

Comment:
@ally-sassman maybe this is the place to mention (in general terms), how to get large videos from contributors. How about adding a comment like this: "If the contributor's video is large, you can have them upload it to our Google team drive" (without being specific about team drive name).